### PR TITLE
Fix lost custom client scope in association queries

### PR DESCRIPTION
**Pull request by einzige**

### CODE REVIEW

File: spec/lib/mavenlink/client_spec.rb [hunk 1]
```diff
@@ -13,6 +13,46 @@
...
+    specify do
+      expect(record.participants.count).to eq(1)
+    end
+
+    specify do
+      expect(record.participants.first).to be_a(Mavenlink::User)
+    end
+
+    it 'saves the client scope' do
+      expect(record.participants.first.client).to eq(client)
```
Comment (einzige) [inline spec/lib/mavenlink/client_spec.rb]: @cantino this is the case: http://stackoverflow.com/questions/29288253/ruby-instance-variable-across-classes-or-a-better-way-to-share-an-object-among-c

**Comment by einzige** [comment]
Updated gem dependecies

esolving dependencies...
Installing i18n 0.7.0 (was 0.6.9)
Installing rake 10.4.2 (was 10.2.2)
Installing minitest 5.6.1 (was 4.7.5)
Installing thread_safe 0.3.5 (was 0.3.2)
Installing json 1.8.2
Installing builder 3.2.2 (was 3.1.4)
Installing awesome_print 1.6.1 (was 1.2.0)
Using coderay 1.1.0
Using diff-lcs 1.2.5
Using multipart-post 2.0.0
Installing hitimes 1.2.2
Using thor 0.19.1
Installing formatador 0.2.5 (was 0.2.4)
Installing rb-fsevent 0.9.5 (was 0.9.4)
Installing lumberjack 1.0.9 (was 1.0.5)
Installing nenv 0.2.0
Installing shellany 0.0.1
Using method_source 0.8.2
Installing slop 3.6.0 (was 3.5.0)
Using rspec-core 2.14.8
Using rspec-mocks 2.14.6
Using bundler 1.9.1
Installing ffi 1.9.8 (was 1.9.3)
Using rspec-expectations 2.14.5
Installing tzinfo 1.2.2 (was 0.3.39)
Installing timers 4.0.1 (was 1.1.0)
Installing faraday 0.9.1 (was 0.9.0)
Installing foreman 0.78.0 (was 0.63.0)
Installing pry 0.10.1 (was 0.9.12.6)
Installing notiffany 0.0.6
Using rspec 2.14.1
Installing rb-inotify 0.9.5 (was 0.9.3)
Installing celluloid 0.16.0 (was 0.15.2)
Installing listen 2.10.0 (was 2.7.1)
Installing activesupport 4.2.1 (was 4.0.4)
Installing activemodel 4.2.1 (was 4.0.4)
Using brainstem-adaptor 0.0.3
Using shoulda-matchers 2.5.0
Using mavenlink 0.0.1 from source at .
Installing guard 2.12.5 (was 2.6.0)
Installing guard-rspec 4.3.1 (was 4.2.8)

**Comment by einzige** [comment]
:green_apple: :yum: :green_apple:

**Comment by cantino** [comment]
Thanks @einzige!

@adambedford, could you review this?

**Comment by adambedford** [comment]
@cantino @einzige Looks good to me.

**Comment by cantino** [comment]
Thanks!

**Pull request merged**